instrument/config.go

```diff
@@ -94,10 +97,16 @@ func (mc *MetricsConfiguration) NewRootScopeReporter(
 		}
 	}
 
+	var sanitizeOpts *tally.SanitizeOptions
+	if mc.MetricSanitization != nil {
+		sanitizeOpts = mc.MetricSanitization.Options()
```

Comment:
nit: rename `Options` to `NewOptions` to be consistent with other config-related constructors.